template/script.sh.erb

```diff
@@ -10,9 +11,9 @@
 %>
 
 # Setup environment
-CODE_SERVER="${CODE_SERVER:-/users/PZS0562/efranz/code-server2.1692-vsc1.39.2-linux-x86_64/code-server}"
+CODE_SERVER="${CODE_SERVER:-/users/PZS0715/msqueo/ondemand/dev/bc_osc_codeserver/code-server-3.1.1-linux-x86_64/code-server}"
```

Comment:
We should improve the context for rendering this to make it easier to access this information, but....

You can get the batch connect app root by adding this ruby code to the script.sh.erb:

```
app_root = BatchConnect::App.from_token(session.token).root.realpath
```

So you could do this for setting up the environment:

    CODE_SERVER="${CODE_SERVER:-<%= BatchConnect::App.from_token(session.token).root.realpath %>/code-server/code-server}"

And then you would update the directions to deploy whatever version of code server you want to the APPROOT/code-server directory (instead of code-server-3.1.1-linux-x86_64; of course code-server-3.1.1-linux-x86_64 is nice to remind you what version is deployed but in that case maybe it is a symlink we add that is code-server to whatever is deployed).

This way when we deploy the app nothing has to change in the code to make it work.